cm4-tool/tool.c

```diff
@@ -16,7 +16,7 @@
 #include <termios.h>
 #include <sys/msg.h>
 #include <sys/threads.h>
-#include <imxrt117x-cm4.h>
+#include <imxrt-multi.h>
```

Comment:
CM4 IDs are not available here (TARGET_IMXRT1170 define is missing).

Reply:
We don't need them anyway